css/properties/gap.json

```diff
@@ -33,10 +33,10 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "14.1"
```

Comment:
Yes! Thanks to @svillar for [closing the gap in flexbox](https://blogs.igalia.com/svillar/2020/10/01/closing-the-gap-in-flexbox/) 🎉